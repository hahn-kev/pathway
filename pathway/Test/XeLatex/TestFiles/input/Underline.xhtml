<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="utf-8" lang="utf-8">
<head>
<title /><link rel="stylesheet" href="Underline.css" type="text/css" /></head>
<body>
<div class="uc">Maybe some of the LaTeX folks here can suggest how this would be integrated properly with LaTeX environments and other parameters. But it may be enough to get you started, at least. </div>
</body>
</html>
