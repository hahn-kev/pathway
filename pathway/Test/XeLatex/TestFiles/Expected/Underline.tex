\documentclass{article} 
\usepackage{mdframed,xcolor}
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}

\begin{document} 
\thispagestyle{empty} 
\font\uc="Times New Roman" at 12pt

\underbar{\uc{Maybe some of the LaTeX folks here can suggest how this would be integrated properly with LaTeX environments and other parameters. But it may be enough to get you started, at least. }}

\end{document}
